src/scenic/syntax/scenic.gram

```diff
@@ -854,6 +854,14 @@ scenic_monitor_def_block:
 
 scenic_monitor_statements[list]: a=statement+ { list(itertools.chain.from_iterable(a)) }
 
+# Override
+# --------
+
+scenic_override_stmt:
+    | 'override' e=primary ss=scenic_specifiers NEWLINE { s.Override(target=e, specifiers=ss) }
```

Comment:
I tried to write a single rule that matches single-line and multiline specifiers, but I couldn't find a way to do it cleanly due to the implementation of Pegen.

```
'override' e=primary ss=scenic_specifiers t=(NEWLINE { [] } | ',' NEWLINE INDENT x=scenic_new_block_body DEDENT { x })
```

Something like this should work, but since `[]` is a falsy value in Python and Pegen depends on Python's truthy check to determine if it has found a matching string, the first case of `t` is never accepted. I can create a custom value that is truthy and convert that to the empty list inside curly braces; however, I figured writing separate rules is cleaner.

Reply:
Agreed, the grammar is more understandable this way.